libs/core/src/mui/index.ts

```diff
@@ -2,4 +2,4 @@
  * Re-export all other components from `@mui/material`
  * which is not implemented by `Sahab Design System`
  * */
-export {};
+export { createSvgIcon } from '@mui/material';
```

Comment:
why did you export `createSvgIcon`.
Here we re-export the components and utils from `mui` which are not manipulated by our design system and use directly in the apps

Reply:
`createSvgIcon` is a `mui` util that we can use to make svgs.